docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -82,28 +58,54 @@ scheduled, but will fail to launch.
 **Warning**: Due to their nature of being _unsafe_, the use of _unsafe_ sysctls
 is at-your-own-risk and can lead to severe problems like wrong behavior of
 containers, resource shortage or complete breakage of a node.
-
-## Enabling Unsafe Sysctls
+{: .warning}
 
 With the warning above in mind, the cluster admin can allow certain _unsafe_
 sysctls for very special situations like e.g. high-performance or real-time
 application tuning. _Unsafe_ sysctls are enabled on a node-by-node basis with a
 flag of the kubelet, e.g.:
 
 ```shell
-$ kubelet --experimental-allowed-unsafe-sysctls 'kernel.msg*,net.ipv4.route.min_pmtu' ...
+$ kubelet --experimental-allowed-unsafe-sysctls \
+  'kernel.msg*,net.ipv4.route.min_pmtu' ...
 ```
+
 For minikube, this can be done via the `extra-config` flag:
 
 ```shell
 $ minikube start --extra-config="kubelet.AllowedUnsafeSysctls=kernel.msg*,net.ipv4.route.min_pmtu"...
 ```
+
 Only _namespaced_ sysctls can be enabled this way.
 
+
 ## Setting Sysctls for a Pod
 
-The sysctl feature is an alpha API in Kubernetes 1.4. Therefore, sysctls are set
-using annotations on pods. They apply to all containers in the same pod.
+A number of sysctls are _namespaced_ in today's Linux kernels. This means that
```

Comment:
Generally reads great, but it looks like there's some lingering errors in the template - missing segments. Scan to the bottom of https://deploy-preview-6808--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/sysctl-cluster/ 
 - there's a `ERROR: You must define a prerequisites block` and a `ERROR: You must define a steps block`